Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+
+
+--
+-- TOC entry 2831 (class 2606 OID 17311)
+-- Name: PSM Section PSM Section_accession_fkey; Type: FK CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."PSM Section"
+    ADD CONSTRAINT "PSM Section_accession_fkey" FOREIGN KEY (accession) REFERENCES "MzTab"."Peptide Section"(accession) NOT VALID;
+
+
+--
+-- TOC entry 2830 (class 2606 OID 17296)
+-- Name: Peptide Section Peptide Section_accession_fkey; Type: FK CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Peptide Section"
+    ADD CONSTRAINT "Peptide Section_accession_fkey" FOREIGN KEY (accession) REFERENCES "MzTab"."Protein section"(accession) NOT VALID;
+
+
+--
+-- TOC entry 2829 (class 2606 OID 17281)
+-- Name: Protein section Protein section_mzTab-ID_REFS_fkey; Type: FK CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Protein section"
+    ADD CONSTRAINT "Protein section_mzTab-ID_REFS_fkey" FOREIGN KEY ("mzTab-ID_REFS") REFERENCES "MzTab"."Metadata_Proteomics"("mzTab-ID") NOT VALID;
+
+
+--
+-- TOC entry 2832 (class 2606 OID 18255)
+-- Name: Small Molecule Evidence (SME) Section Small Molecule Evidence (SME) Section_SME_ID_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Evidence (SME) Section"
+    ADD CONSTRAINT "Small Molecule Evidence (SME) Section_SME_ID_fkey" FOREIGN KEY ("SME_ID") REFERENCES "MzTab-M"."Small Molecule Feature (SMF) Section"("SME_ID_REFS") NOT VALID;
+
+
+--
+-- TOC entry 2833 (class 2606 OID 18248)
+-- Name: Small Molecule Feature (SMF) Section Small Molecule Feature (SMF) Section_SMF_ID_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Feature (SMF) Section"
+    ADD CONSTRAINT "Small Molecule Feature (SMF) Section_SMF_ID_fkey" FOREIGN KEY ("SMF_ID") REFERENCES "MzTab-M"."Small Molecule Section"("SMF_ID_REFS") NOT VALID;
+
+
+--
+-- TOC entry 2828 (class 2606 OID 18261)
+-- Name: Small Molecule Section Small Molecule Section_SMF_ID_REFS_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Section"
+    ADD CONSTRAINT "Small Molecule Section_SMF_ID_REFS_fkey" FOREIGN KEY ("SMF_ID_REFS") REFERENCES "MzTab-M"."Small Molecule Feature (SMF) Section"("SMF_ID") NOT VALID;
+
+
+-- Completed on 2020-10-30 17:54:26
+
+--
+-- PostgreSQL database dump complete
+--
+
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 12.3
+-- Dumped by pg_dump version 12.3
+
+-- Started on 2020-10-30 17:12:34
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+DROP DATABASE "MFA analysis";
```

Comment:
As above, Let's change the name of the database.

Reply:
done